Client/Configuration/Fennec.xcconfig

```diff
@@ -12,4 +12,9 @@ INCLUDE_SETTINGS_BUNDLE = YES
 
 // Adjust SDK Variables
 ADJUST_ENVIRONMENT =
-ADJUST_APP_TOKEN = 
+ADJUST_APP_TOKEN =
+
+// Leanplum SDK Variables
+LEANPLUM_ENVIRONMENT = development
+LEANPLUM_APP_ID = 
+LEANPLUM_KEY =
```

Comment:
@st3fan just a reminder here - I took out the API keys so where can I put them now?